modules/wri_node/wri_node.module

```diff
@@ -60,20 +61,39 @@ function wri_node_preprocess_wri_page_hero(&$context) {
 function wri_node_preprocess_layout__card(&$variables) {
   $variables['default_image'] = [];
   if (Drupal::config('wri_node.settings')->get('use_fallback_image')) {
-    if (isset($variables["content"]['tag']["field_primary_topic"][0]["#title"])) {
-      $image_url = \Drupal::service('extension.list.module')
-        ->getPath('wri_node') . '/images/' . strtolower(str_replace(' ', '-', $variables["content"]['tag']["field_primary_topic"][0]["#title"])) . '.jpg';
-      if (!file_exists($image_url)) {
-        $image_url = \Drupal::service('extension.list.module')
-          ->getPath('wri_node') . '/images/default.jpg';
+    $node = $variables['content']['#node'];
+    $topic = $node->field_primary_topic->target_id;
+    $image_url = '';
+    $theme = 'image';
+    if ($topic) {
+      // Get default image form term.
+      $term = Term::load($topic);
+      if ($term) {
+        if ($term->field_default_image && $term->field_default_image->target_id) {
```

Comment:
I added an additional check here for sites that may not have the default image field yet.